src/rgw/rgw_sal_motr.cc

```diff
@@ -41,6 +41,7 @@ extern "C" {
 #include "rgw_bucket.h"
 #include "rgw_quota.h"
 #include "motr/addb/rgw_addb.h"
+#include "rgw_tag_s3.h"
```

Comment:
Can we remove this as we are not using `RGWObjTagging_S3` class ?

Reply:
Yes, Done